packages/feedback-react/src/BaseFeedback.tsx

```diff
@@ -24,30 +28,37 @@ export interface BaseFeedbackProps {
     headingLevel?: "h1" | "h2" | "h3" | "h4" | "h5" | "p";
 }
 
+interface Props extends BaseFeedbackProps {
+    content: ReactNode;
+}
+
 export const FeedbackContext = createContext<{
     options: FeedbackValue[];
     value?: FeedbackValue;
     setValue: (next: FeedbackValue) => void;
 }>({ options: [], setValue: () => undefined });
 
-export const BaseFeedback: FC<BaseFeedbackProps> = ({
+export const BaseFeedback: FC<Props> = ({
     label,
     onSubmit,
     description,
+    renderCustomSuccess,
     successMessage = "Det hjelper oss i arbeidet med å forbedre våre løsninger",
     successTitle = "Takk for tilbakemeldingen!",
     callToActionText = "Send inn tilbakemelding",
     showTextArea = true,
     textAreaLabel = "Tips oss gjerne om hva vi kan forbedre",
     textAreaHelpLabel = "",
     className = "",
-    feedbackOptions = [1, 2, 3, 4, 5],
+    feedbackOptions = [VERY_UNHAPPY, UNHAPPY, NEUTRAL, HAPPY, VERY_HAPPY],
```

Comment:
Hvorfor importeres disse en og en og spesifiseres en og en? Hadde ikke. det vært lettere med et enum/objekt/array som ble eksportert istedet? 🤔

Reply:
Tenkte de en og en, for at det skulle være enkelt å bruke på klientside hvis du feks (som vi i hund) bare vil ha "UNHAPPY" og "HAPPY".